pkgs/top-level/all-packages.nix

```diff
@@ -426,6 +426,8 @@ with pkgs;
 
   cryptowatch-desktop = callPackage ../applications/finance/cryptowatch { };
 
+  databricks-sql-cli = with python3Packages; toPythonApplication databricks-sql-cli;
```

Comment:
We see that other python cli tools (such as httpie) are both included as python package and application.

Reply:
If this program has only a cli and can't be imported as a python module then it should not go into pythonPackages.

IIRC httpie is only in pythonPackages because the way its plugins system works and it was easier this way.

Reply:
Thanks @SuperSandro2000 for you clarification. Have now [changed](https://github.com/NixOS/nixpkgs/compare/7ebdb6e132ced29a3e27cf3283b71228d96e9a0e..aaabe7f19d7cbcba20bb80175b538f87e4bfc8c0) `databricks-sql-cli` to only be an application.